packages/nx/src/executors/run-commands/run-commands.impl.ts

```diff
@@ -292,14 +292,24 @@ export function interpolateArgsIntoCommand(
     const regex = /{args\.([^}]+)}/g;
     return command.replace(regex, (_, group: string) => opts.parsedArgs[group]);
   } else if (forwardAllArgs) {
+    const filteredUnparsed = filterUnparsed(opts.__unparsed__);
     return `${command}${
-      opts.__unparsed__.length > 0 ? ' ' + opts.__unparsed__.join(' ') : ''
+      filteredUnparsed.length > 0 ? ' ' + filteredUnparsed.join(' ') : ''
     }`;
   } else {
     return command;
   }
 }
 
+function filterUnparsed(unparsed: string[]) {
```

Comment:
I think the `run-commands` executor is too high up for this logic.

I think `__unparsed__` wouldn't include CLI arguments that we have actually parsed such as the properties of the builder.

Could you explore if there is a change we could make where `__unparsed__` is defined?

Reply:
Thanks for your feedback!
I'll have a look!